src/components/Browse.tsx

```diff
@@ -1,37 +1,30 @@
 import React from "react";
-import { useNavigate } from "react-router-dom";
-import { useSelector } from "react-redux";
-import { RootState } from "../utils/appStore";
-import { signOut } from "firebase/auth";
-import { auth } from "../utils/firebase";
+import { useNowPlayingMovies } from "../hooks/useNowPlayingMovies";
+import BrowseNavbar from "./BrowseNavbar";
+import MainContainer from "./MainContainer";
+import SecondaryContainer from "./SecondaryContainer";
+import { usePopularMovies } from "../hooks/usePopularMovies";
+import { useTopRatedMovies } from "../hooks/useTopRatedMovies";
+import { useUpcomingMovies } from "../hooks/useUpcomingMovies";
+import { useAiringTodayTVshows } from "../hooks/useAiringTodayTVshows";
+import { usePopularTVshows } from "../hooks/usePopularTVshows";
+import { useTopRatedTVshows } from "../hooks/useTopRatedTVshows";
 
 const Browse: React.FC = () => {
-  const cartUser = useSelector((state: RootState) => state.user);
-
-  console.log("Browse Page Rendering ....");
-  const navigate = useNavigate();
-
-  function handleSignOut() {
-    signOut(auth).then(() => {
-      navigate("/login");
-    }).catch((error) => {
-      console.log("error=",error);
-    });
-  }
+  useNowPlayingMovies();
```

Comment:
Good to see hooks used here to separate the fetching logic.
Extras (not required): Check out react-query for API fetching, you will love it once you understand it

Reply:
done